src/Components/ProgressiveOnboarding/ProgressiveOnboardingFollowArtist.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC, useCallback, useEffect } from "react"
+import { Text } from "@artsy/palette"
+import { ProgressiveOnboardingPopover } from "Components/ProgressiveOnboarding/ProgressiveOnboardingPopover"
+import { useProgressiveOnboarding } from "Components/ProgressiveOnboarding/ProgressiveOnboardingContext"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { useSystemContext } from "System"
+import { ProgressiveOnboardingFollowArtistQuery } from "__generated__/ProgressiveOnboardingFollowArtistQuery.graphql"
+import { ProgressiveOnboardingFollowArtist_me$data } from "__generated__/ProgressiveOnboardingFollowArtist_me.graphql"
+
+export const PROGRESSIVE_ONBOARDING_FOLLOW_ARTIST = "follow-artist"
+
+interface ProgressiveOnboardingFollowArtistProps {
+  me: ProgressiveOnboardingFollowArtist_me$data
+}
+
+export const ProgressiveOnboardingFollowArtist: FC<ProgressiveOnboardingFollowArtistProps> = ({
+  me,
+  children,
+}) => {
+  const counts = me.counts || { followedArtists: 0 }
+
+  const { dismiss, isDismissed, enabled } = useProgressiveOnboarding()
+
+  const handleClose = useCallback(() => {
+    dismiss(PROGRESSIVE_ONBOARDING_FOLLOW_ARTIST)
+  }, [dismiss])
+
+  useEffect(() => {
+    if (counts.followedArtists === 0) return
```

Comment:
Do we want to only display this if you haven't followed any artists? I'm also a little unclear on whether or not we want to tie this to account creation date.